package.json

```diff
@@ -1955,6 +1963,12 @@
         "command": "calva.showResultOutputDestination",
         "title": "Show/Open the result output destination"
       },
+      {
+        "category": "Calva",
+        "command": "calva.showReplOutputWebview",
+        "title": "Show/Open the REPL output webview",
+        "enablement": "calva:connected"
```

Comment:
I have started to think we shouldn't disable commands as often as we do. It can create confusion as to why something is not there, not doing anything, and so on. And opening the output view makes total sense when the repl isn't connected.

Reply:
Sounds good. I actually think similarly, but was just copying the other similar commands for now. I'll remove this enablement entry though.